modules/govuk/manifests/apps/content_data_admin.pp

```diff
@@ -50,6 +50,11 @@
 # [*google_tag_manager_auth*]
 #   The identifier of an environment for Google Tag Manager
 #
+# [*read_timeout*]
+# Configure the amount of time the nginx proxy vhost will wait for the
+# backing app before it sends the client a 504. We override the default 15 seconds
+# to 60.
+#
```

Comment:
This comment is a bit misleading, as this class doesn't have a `read_timeout` option. It might be good to remove this to keep the commends and code consistent.